api/v1alpha3/types.go

```diff
@@ -250,18 +250,12 @@ var (
 
 // Bastion represents basic information about the bastion node
 type Bastion struct {
-	//+optional
+	// optional
```

Comment:
Does it still work like this? I would have thought it hast to look like `//+optional`?